Sorting/merge.cpp

```diff
@@ -2,9 +2,74 @@
 
 /**
  * Sort the array using merge sort algorithm.
- * 
+ *
  * Inputs: null
  */
-void Sort::MergeSort(){
-    std::cout<<"[TODO @ Sorting/merge.cpp] MergeSort() needs to be implemented.\n";
-}
\ No newline at end of file
+ std::vector<int> arr;
+ void Merge(int low, int mid, int high)
```

Comment:
Good work, but you need to do this in a separate PR. This is not type-1. Refer README.